external/netstandard/netstandard1.x/project.json.template

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "NETStandard.Library": "2.0.0-beta-24919-01",
+        "System.Diagnostics.Contracts": "4.3.0"
```

Comment:
What is the reason for the different template for 1.x? The only difference appears to be this System.Diagnostics.Contracts package. Are you working around some issue?